hopp/simulation/technologies/sites/site_info.py

```diff
@@ -66,6 +67,8 @@ class SiteInfo(BaseClass):
     hub_height: hopp_float_type = field(default=97., converter=hopp_float_type)
     capacity_hours: NDArray = field(default=[], converter=converter(bool))
     desired_schedule: NDArrayFloat = field(default=[], converter=converter())
+    curtailment_value_type: str = field(default="desired_schedule", validator=contains(["grid", "desired_schedule"]))
```

Comment:
To be consistent here, we might want to make "grid" the default and have a test to see if a desired schedule is given, otherwise if it's set to desired_schedule and site.py doesn't have that variable, you'll get an error

Reply:
Never mind, I just realized it's in the desired_schedule loop so you won't get an error. I still think we should put grid as the default though since it seems to be what people prefer

Reply:
Okay I changed it to default to "grid"